plugins/webp-uploads/tests/data/class-testcase.php

```diff
@@ -137,6 +137,6 @@ public function set_image_output_type( string $format ): void {
 	 * Opt into picture element output.
 	 */
 	public function opt_in_to_picture_element(): void {
-		add_filter( 'wp_content_img_tag', 'webp_uploads_wrap_image_in_picture', 10, 3 );
+		webp_uploads_init();
```

Comment:
Shouldn't this also make sure that it removes filters that `webp_uploads_init()` adds?

```suggestion
		remove_filter( 'wp_content_img_tag', 'webp_uploads_wrap_image_in_picture', 10, 3 );
		remove_filter( 'the_content', 'webp_uploads_update_image_references', 13 ); // Run after wp_filter_content_tags.
		webp_uploads_init();
```

Reply:
Thanks. When it runs at first, it will add the `the_content` filter, but this is a nice addition to avoid any issues in tests.

PHPStan reports `Function remove_filter invoked with 4 parameters, 2-3 required.` so I have removed the `$accepted_args` from the `wp_content_img_tag` filter as it is not needed for `remove_filter`.